src/main/java/org/cyclops/everlastingabilities/client/gui/GuiAbilityContainer.java

```diff
@@ -266,6 +267,7 @@ protected void drawXp(int x, int y) {
 
     private void drawAbilities(int x, int y, List<Ability> abilities, int startIndex, int playerXp,
                                int currentSelectedIndex, int mouseX, int mouseY, boolean canEdit) {
+        Collections.sort(abilities);
```

Comment:
Note to self: if the number of abilities in a player/container would become significantly larger, sorting each render tick might introduce too much overhead. But this isn't a problem for now, as the number of abilities is very low.

Reply:
Indeed, your solution is fine.

If performance issues would arise. We could still sort the abilities when the gui is opened (or the abilities are updated). As the language shouldn't be able to change while the gui is open.